lib/elixir/lib/range.ex

```diff
@@ -392,8 +394,17 @@ defmodule Range do
 
   @doc """
   Converts a range to a list.
+
+  ## Examples
+
+      iex> Range.to_list(0..5)
+      [0, 1, 2, 3, 4, 5]
+      iex> Range.to_list(-3..0)
+      [-3, -2, -1, 0]
+
   """
   @doc since: "1.15.0"
+  @spec to_list(t) :: list
```

Comment:
I think we can be even more precise:

```suggestion
  @spec to_list(t) :: list(integer)
```